### PR TITLE
Allow env vars to be passed to tests

### PR DESCRIPTION
**Pull request by danmcp**

### CODE REVIEW

**Comment by danmcp** [comment]
@deads2k @stevekuznetsov FYI

**Comment by deads2k** [comment]
I can't read ruby.  Does this allow `--env="one=a" --env="two=b"`?

@liggitt ptal.

File: lib/vagrant-openshift/command/test_origin.rb [hunk 1]
```diff
@@ -74,6 +74,11 @@ def execute
             o.on("-r","--image-registry", String, "Image registry to configure tests with") do |f|
               options[:image_registry] = f
             end
+
+            o.on("", "--env ENV=VALUE", String, "Environment variable to execute tests with") do |f|
```
Comment (liggitt) [inline lib/vagrant-openshift/command/test_origin.rb]: `env` vs `:envs`?

File: lib/vagrant-openshift/command/test_origin.rb [hunk 2]
```diff
@@ -74,6 +74,11 @@ def execute
...
+              options[:envs] = [] unless options[:envs]
+              options[:envs] << f
```
Comment (liggitt) [inline lib/vagrant-openshift/command/test_origin.rb]: this appends for each --env arg?

Comment (deads2k) [inline lib/vagrant-openshift/command/test_origin.rb]: > this appends for each --env arg?

Should I be concerned that three lines of ruby appears to be difficult to read?

Comment (stevekuznetsov) [inline lib/vagrant-openshift/command/test_origin.rb]:
```rb
options[:envs] = [] unless options[:envs]
```
what's hard to read about that?

File: lib/vagrant-openshift/command/test_origin.rb (hunk 1 above)
Comment (danmcp) [inline lib/vagrant-openshift/command/test_origin.rb]: You are passing 1 env at a time.  But the entry in options is storing all of the envs.

**Comment by danmcp** [comment]
@deads2k 

> I can't read ruby. Does this allow --env="one=a" --env="two=b"?

It does


@liggitt 

> this appends for each --env arg?

It does


@deads2k 

> Should I be concerned that three lines of ruby appears to be difficult to read?

unclear



@stevekuznetsov 

> what's hard to read about that?

Squint a little more

**Comment by stevekuznetsov** [comment]
@danmcp sorry I forgot this `</sarcasm>`

@bparees @danmcp is there anything else this needs? I can do the update job and update jenkins to use this so jUnit gets produced when it needs to

**Comment by danmcp** [comment]
@stevekuznetsov Functionally I think it's ok.  I am debating error handling and whether we care about any attack vectors.

**Pull request merged**